### PR TITLE
MGDCTRS-1997: upgrade rhoc CLI to leverage the operator upgrade command.

### PR DESCRIPTION
**Pull request by valdar**

### CODE REVIEW

**Review by rinaldodev — APPROVED** [review]
We just need to keep in mind that this is not completetly supported in the fleetshard yet.

**Comment by valdar** [comment]
> We just need to keep in mind that this is not completetly supported in the fleetshard yet.

We can merge this one after the other PR are released, or merge this one and do a release of the CLI after the other PR are released. In any case the call to `rhoc deployments update --revision xxx` would work, while `rhoc deployments update --operator-id xxx` will do nothing so I think we are safe either way.

**Pull request merged**
